plugins/backstage-opa-backend/src/api/EntityCheckerApi.ts

```diff
@@ -0,0 +1,99 @@
+import fetch from 'node-fetch';
+import { LoggerService } from '@backstage/backend-plugin-api';
+import { Entity } from '@backstage/catalog-model';
+
+export interface EntityCheckerApi {
+  checkEntity(options: checkEntityOptions): Promise<OpaEntityCheckResult>;
+}
+
+export type checkEntityOptions = {
+  entityMetadata: Entity;
+};
+
+export type EntityCheckerConfig = {
+  logger: LoggerService;
+  opaBaseUrl: string | undefined;
```

Comment:
One thing I've been considering doing more around the OPA plugins is just defaulting the opaBaseUrl to `localhost:8181` if its missing in the config, kind of like [this](https://github.com/Parsifal-M/backstage-opa-plugins/blob/main/packages/opa-authz/src/api/opaClient.ts#L22)

wdyt?